Source/WebKit/Platform/spi/Cocoa/PDFKitSPI.h

```diff
@@ -179,6 +171,14 @@ typedef NS_ENUM(NSUInteger, PDFSelectionGranularity);
 
 #endif
 
+#if HAVE(PDFKIT_WITH_NEXT_ACTIONS) && __has_include(<PDFKit/PDFActionPriv.h>)
+#import <PDFKit/PDFActionPriv.h>
+#else
+@interface PDFAction (Staging_132413786)
```

Comment:
Hm... I would've expected 132413786 to be a bug in PDFKit.

Reply:
Right, of course. I'll change this to rdar://122820963

Reply:
Although, now that I think about it, currently this is less about staging the change and more about having the declaration available for public SDK builds. Maybe I'll just rename the category to `SPI`.